src/keyboard.c

```diff
@@ -14,6 +14,8 @@ See the file LICENSE for details.
 
 #define KEY_INVALID 0127
 
+#define W_SUBSTITUTE -1
```

Comment:
When we switch the key invalid value, it will just stomp on another valid ascii value. I experimented with several values, but the mapping was odd, and it's hard to find the correlation of key mapping to ascii values. 
I agree it's sketchy, but I also think it's akin to putting plywood over that sketchy hole of memory